streams/src/main/java/org/apache/kafka/streams/kstream/internals/graph/ProcessorNode.java

```diff
@@ -26,9 +26,8 @@
  * Used to represent any type of stateless operation:
  *
  * map, mapValues, flatMap, flatMapValues, filter, filterNot, branch
- *
  */
-public class StatelessProcessorNode<K, V> extends StreamsGraphNode {
+public class ProcessorNode<K, V> extends StreamsGraphNode {
```

Comment:
I thought it was a little weird previously that `StatefulProcessorNode` extends `StatelessProcessorNode`, since subclassing is generally an "is a" relationship. So it was previously saying "StatefuProcessorNode is a StatelessProcessorNode", which is silly of course.

Maybe we can call it `ProcessorGraphNode` to avoid a collision?

Reply:
ack